CliFx/Domain/HelpTextWriter.cs

```diff
@@ -246,21 +246,45 @@ void RenderOptions()
 
                     RenderColumnIndent();
 
-                    // Description
-                    if (!string.IsNullOrWhiteSpace(option.Description))
-                    {
-                        Render(option.Description);
-                        Render(" ");
-                    }
+                    RenderOptionDescription(option);                    
+
+                    RenderNewLine();
+                }
+            }
+
+            void RenderOptionDescription(CommandOptionSchema option)
+            {
+                // Description
+                if (!string.IsNullOrWhiteSpace(option.Description))
```

Comment:
Looks like valid values and env variable name is only rendered if the description is set. I don't think that's right.

Reply:
Fixed. I had accidentally moved it in the `RenderOptionDescription()` method before which has been removed as of this latest commit.